File: JavaScript/5kyu/NumberOfTtrailingZeros.js
function zeros_kata_1(n) {
  var fives = 0;
  while (n > 0) {
    n = Math.floor(n / 5);
    fives += n;
  }
  return fives;
}

function zeros(n) {
  return n / 5 < 1 ? 0 : Math.floor(n / 5) + zeros(n / 5);
}

console.log(zeros(0));
console.log(zeros(5));
console.log(zeros(6));
console.log(zeros(30));

File: Python/5kyu/NumberOfTtrailingZeros.py
from unittest import result


def zeros_kata_1(n):
    result = 0
    while(n > 0):
        n = int(n / 5)
        result += n

    return result


def zeros(n):
    x = int(n/5)
    return x + zeros(x) if x else 0


print(zeros(100))
print(zeros(6))
print(zeros(30))
